src/player_team.ts

```diff
@@ -997,7 +997,22 @@ class Team {
       mults[i].base = pings[i].damage;
     }
 
-    let healing = 0;
+    // Apply poison damage.
+    let poison = 0;
+    for (const c of 'pm') {
+      const attr = COLORS.indexOf(c) as Attribute;
+      for (const combo of comboContainer.combos[c]) {
+        let multiplier = 0;
+        if (c == 'p') {
+          multiplier = 0.2 + (combo.count - 3) * 0.05;
+        } else if (c == 'm') {
+          multiplier = 0.5 + (combo.count - 3) * 0.125;
```

Comment:
This can be simplified to (combo.count + 1) * 0.125

Reply:
See above.